app/views/user_mailer/monthly_owner_wrapup.html.slim

```diff
@@ -0,0 +1,44 @@
+p Hello #{@owner.display_name},
+
+p Another month has passed.  Give your collaborators some love and encourage them to come back by sending a monthly status email.  The email could include:
+ul
+  li What they've accomplished this month
+  li What you'll be using the transcribed material for
+  li call to action - "Know anyone else who would be interested in working on this collection?  Invite them to join in on the fun."
+  li Here's an example of a monthly email: 
+  span =link_to "http://content.fromthepage.com/crowdsourcing-project-newsletter", 'http://content.fromthepage.com/crowdsourcing-project-newsletter', target: '_blank', rel: 'nofollow'
+p Here are the stats for all of your collections in the past month:
+
+table
+  tr
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
```

Comment:
I'm wondering how hard it would be to do "Total Works" and "Works Complete"?  That would be a nice stats to have.

Reply:
Not hard, it turns out - I added the statistic to make it available.  How do you want that laid out in the table, though?

Reply:
I'd like a "Total Works" and "Completed Works"

If we drop the notes count, we could do "total works" "completed works" "pages" "collaborators" on the first line and keep the rest as it is.